lms/djangoapps/learner_home/serializers.py

```diff
@@ -301,21 +301,18 @@ def get_availableDate(self, enrollment):
         course_overview = enrollment.course_overview
         available_date = course_overview.certificate_available_date
```

Comment:
🤔 Should we change the default of this now that the old way to do things has been removed? Or maybe it should be an `else`/default case as part of the if/elif?

I know we've had issues with holding on to stale data in the past and I'm just trying to think of this from a defensive standpoint. If the display behavior is "immediately upon passing" or the course is self-paced, the cert available date should be None.

Reply:
good point, change made.